com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDDefaultSettingsEditor.cs

```diff
@@ -0,0 +1,35 @@
+using UnityEngine.Rendering.HighDefinition;
+using UnityEngine;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    [CustomEditor(typeof(HDDefaultSettings))]
+    [CanEditMultipleObjects]
+    sealed class HDDefaultSettingsEditor : Editor
+    {
+        SerializedHDDefaultSettings m_SerializedHDDefaultSettings;
+
+        internal bool largeLabelWidth = true;
+
+        void OnEnable()
+        {
+            m_SerializedHDDefaultSettings = new SerializedHDDefaultSettings(serializedObject);
+        }
+
+        public override void OnInspectorGUI()
+        {
+            var serialized = m_SerializedHDDefaultSettings;
+
+            serialized.serializedObject.Update();
+
+            // In the quality window use more space for the labels
+            if (!largeLabelWidth)
+                EditorGUIUtility.labelWidth *= 2;
+            DefaultSettingsPanelIMGUI.Inspector.Draw(serialized, this);
+            if (!largeLabelWidth)
+                EditorGUIUtility.labelWidth *= 0.5f;
+
+            serialized.serializedObject.ApplyModifiedProperties();
```

Comment:
ApplyModifiedProperties() should only be called when there is a change in the UI, you can add a change check to the code below.

(This function is quiet expansive and depending on the content of the serialized data can re-create the whole class every time you call it :) )